sceptre/synapsedev/config/prod/snowflake-access.yaml

```diff
@@ -0,0 +1,9 @@
+template:
+  type: http
+  url: https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.7.8/IAM/snowflake-bucket-access.yaml
+stack_name: snowflake-access.yaml
+parameters:
+  BucketArn: arn:aws:s3:::dev.datawarehouse.sagebase.org
```

Comment:
this makes me wonder why the role for this bucket isn't created in the same template that created the bucket?

Reply:
Interesting thought, the goal was for org-infra to adopt resources already deployed into the with a shared template (with the benefit that the template can be used for to give Snowflake access to any bucket/prefix in the org)